fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BrokerLoadJob.java

```diff
@@ -346,6 +346,16 @@ private void updateLoadingStatus(BrokerLoadingTaskAttachment attachment) {
         }
     }
 
+    @Override
+    public void updateProgress(Long beId, TUniqueId loadId, TUniqueId fragmentId, long scannedRows,
+                               long scannedBytes, boolean isDone) {
+        super.updateProgress(beId, loadId, fragmentId, scannedRows, scannedBytes, isDone);
+        progress = (int) ((double) loadStatistic.getLoadBytes() / loadStatistic.totalFileSizeB * 100);
```

Comment:
This may always larger than 100, when loading parquet or orc format files.

Reply:
yes， so I force made the max is 99